invenio/modules/deposit/static/css/deposit/form.css

```diff
@@ -64,6 +64,42 @@ span.ui-icon.ui-icon-circle-triangle-e {
   display:none;
 }
 
+.hide-on-collapsed{
+	display:block
+}
+
+.panel-toggle.collapsed .hide-on-collapsed,.show-on-collapsed{
+	display:none
+}
+
+.panel-toggle.collapsed .show-on-collapsed{
+	display:block
+}
+
+.field-without-label {
+  padding: 0 20px;
```

Comment:
which `variables.less`? A new one? If so: the file contains a bunch of variables that do not really belong together and that are for distinct purposes. Not really sure if it's worth to find some forced variables here. We do not have any common space/distance calculation.